melview.py

```diff
@@ -345,6 +356,11 @@ def get_mode(self):
 
         return 'Auto'
 
+    def get_zone(self, zoneid):
+        return next((zone for zone in self.get_zones() if zone.id == zoneid), None)
+
+    def get_zones(self):
```

Comment:
This can just return `self._zones.values()`.

Reply:
done!